Arrowgene.Ddon.GameServer/Characters/CharacterManager.cs

```diff
@@ -13,6 +13,9 @@ public class CharacterManager
 
         public static readonly uint BASE_HEALTH = 760U;
         public static readonly uint BASE_STAMINA = 450U;
+        public static readonly uint BBM_BASE_HEALTH = 990U;
+        public static readonly uint BBM_BASE_STAMINA = 589U;
```

Comment:
Having trouble to confirm what the stamina value should be. When Equipping gear found in videos, this is the value that shows in my client UI. I have not seen a video which shows the base stamina when first making a character though.